traffic_portal/test/integration/config.json

```diff
@@ -29,8 +29,8 @@
     "stopSpecOnExpectationFailure": true
   },
   "params": {
-    "apiUrl": " https://localhost:443/api/4.0",
-    "baseUrl": "https://localhost",
+    "apiUrl": " https://localhost:444/api/4.0",
+    "baseUrl": "https://localhost:444",
```

Comment:
CiaB exposes TP on port 443, not 444 - did you mean to change this?

Reply:
Nope